build-support/pants_venv

```diff
@@ -34,19 +34,22 @@ function create_venv() {
 }
 
 function ensure_gcc() {
-  GCC_VERSION=$(gcc -v 2>&1)
-  if (( $? != 0 )); then
+  if ! gcc -v &>/dev/null; then
```

Comment:
Shellcheck taught me a new idiom!

```bash
if ! GCC_VERSION="$(gcc -v 2>&1)"; then
```

Then you can reference this variable later like normal.

Reply:
Done!